server/src/main/java/com/octopus/teamcity/opentelemetry/server/helpers/OTELHelperImpl.java

```diff
@@ -41,7 +43,7 @@ public OTELHelper(Map<String, String> headers, String exporterEndpoint) {
         this.openTelemetry = OpenTelemetrySdk.builder()
                 .setTracerProvider(sdkTracerProvider)
                 .setPropagators(ContextPropagators.create(W3CTraceContextPropagator.getInstance()))
-                .buildAndRegisterGlobal();
+                .build();
```

Comment:
Now we've got lots of these, we don't want to register global anymore.

Reply:
I don't think we want lots of these? I think this will affect greatly its ability to accurately cross refereance builds to their otel data and exporting to honeycomb? Did we need to have multiple of these for the new implementation?

Reply:
Yeah, given each project can have it's own destination and settings, we "need" a different one for each build.
If we think it's going to be bad, then we could do a "one per combination of settings" thing 🤔